apps/blockchain/lib/blockchain/genesis.ex

```diff
@@ -45,7 +46,7 @@ defmodule Blockchain.Genesis do
           extra_data: "55555555555555555555555555555555",
           gas_limit: 16777216,
           parent_hash: <<0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0>>,
-          state_root: <<33, 123, 11, 188, 251, 114, 226, 213, 126, 40, 243, 60, 179, 97, 185, 152, 53, 19, 23, 119, 85, 220, 63, 51, 206, 62, 112, 34, 237, 98, 183, 123>>,
+          state_root: <<232, 107, 69, 139, 178, 253, 233, 218, 130, 174, 216, 128, 173, 120, 27, 24, 193, 116, 237, 198, 52, 187, 160, 193, 55, 110, 174, 93, 47, 62, 189, 214>>,
```

Comment:
Funny that the state root changed. Is this expected?

Reply:
my bad. it was fixed with  remaining genesis tests